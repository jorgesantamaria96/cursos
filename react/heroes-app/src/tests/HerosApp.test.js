import { mount } from "enzyme";
import { HerosApp } from "../HerosApp";

describe("Pruebas en el component <HerosApp />", () => {
  const wrapper = mount(<HerosApp />);

  test("debe de mostrar el componente correctamente", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
