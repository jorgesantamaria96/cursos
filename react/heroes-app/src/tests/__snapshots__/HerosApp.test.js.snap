// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el component <HerosApp /> debe de mostrar el componente correctamente 1`] = `
<div>
  <div
    className="container mt-5"
  >
    <h1>
      LoginScreen
    </h1>
    <hr />
    <button
      className="btn btn-primary"
      onClick={[Function]}
    >
      Login
    </button>
  </div>
</div>
`;
